src/loops/simple_programs_with_repitition/easy/q6/CountItUp.java

```diff
@@ -0,0 +1,25 @@
+package loops.simple_programs_with_repitition.easy.q6;
+
+/*
+Write a program that will ask the user for a sentence and then a character and 
```

Comment:
This question needs to explain whether upper and lower case of the same character should count as the same - I added the bad question label for you to bring this up in question report later